src/main/resources/sql_mockup/004-insert-shift.sql

```diff
@@ -3,26 +3,26 @@
 --changeset woloapp:004
 
 INSERT INTO shift ( address_to_event_id, start_time, end_time, date, capacity, is_leader_required, required_min_age, directions) VALUES
```

Comment:
Aplikacja nie jest na produkcji, a to są dane mockowe. Natomiast, gdyby to były dane niezbędne na produkcji z jakichś powodów to rozwiązaniem byłby kolejny plik, który zrobiłby `UPDATE TABLE ...`